### PR TITLE
I315

### LINKED ISSUE #315: Change "7-13" checkbox to radio
There are two different parental consent forms, one for children 0-7 and one for children 7-13.  Would be good to change this to a radio button providing these two options

### PR DESCRIPTION
**Pull request by adamrp**
Fixes #315. Note that after merging this, the ag_human_survey table must be changed so that the field called is_7_to_13 is renamed to juvenile_age:

alter table ag_human_survey rename column is_7_to_13 to juvenile_age

### CODE REVIEW

**Comment by adamrp** [comment]
This code is currently deployed on webdev, so please test it out (and keep an eye on the americangut gmail account when you try to create a juvenile participant that isn't on the bruce waynes list). For testing purposes, I added three juvenile participants to the bruce waynes list for the test account: Lil Adam, Lil Doug, and Lil Yoshiki.

**Comment by douginator2000** [comment]
Adam, can you figure out why this is not mergeable and update? Thanks!

**Comment by adamrp** [comment]
Will do!

**Comment by adamrp** [comment]
Conflict resolved, merge away

File: www/americangut/new_participant.psp [hunk 1]
```diff
@@ -142,6 +144,9 @@ To withdraw from the study send email to the <a href="mailto:info@americangut.or
         document.getElementById("parent_1_name").disabled = true
         document.getElementById("parent_2_name").disabled = true
         document.getElementById("deceased_parent").disabled = true
+        document.getElementById("juvenile_age_less_than_7").disabled = true
```
Comment (ElDeveloper) [inline www/americangut/new_participant.psp:147]: Any reason why this line is repeated?

Comment (adamrp) [inline www/americangut/new_participant.psp:147]: it's not :)

Comment (ElDeveloper) [inline www/americangut/new_participant.psp:147]: ah! I see now. Sorry, it's friday and it's getting late :/

**Comment by ElDeveloper** [comment]
:+1:

**Pull request merged**